features/unattached_commands.feature

```diff
@@ -276,6 +276,31 @@ Feature: Command behaviour when unattached
             .*\{ apt update && apt install --only-upgrade -y screen \}.*
             .*✔.* USN-4747-2 is resolved.
             """
+        When I run `apt-get install -y screen=4.1.0~20120320gitdb59704-9 --force-yes` with sudo
+        And I run `ua disable esm-infra` with sudo
+        And I fix `USN-4747-2` by enabling required service
+        Then stdout matches regexp:
+            """
+            USN-4747-2: GNU Screen vulnerability
+            Found CVEs: CVE-2021-26937
+            https://ubuntu.com/security/CVE-2021-26937
+            1 affected package is installed: screen
+            \(1/1\) screen:
+            A fix is available in UA Infra.
+            The update is not installed because this system does not have
+            esm-infra enabled.
+
+            Choose: \[E\]nable esm-infra \[C\]ancel
+            > .*\{ ua enable esm-infra \}.*
+            One moment, checking your subscription first
```

Comment:
Do we need to check their subscription first? 

If they're not attached we'd show the enter your token attach window

if they're not entitled, we'd show they're not entitled and error

This line feels false?

Reply:
When we are running any action inside `ua fix` (`enable`, `attach`, ...) we are actually running then just like an user would. The line you are seeing is the result of running `ua enable`, which always checks the subscription first before enabling a service. With that said, I do agree that this line is not that useful here. I will see if I can easily remove it

Reply:
Ok, thanks appreciate it.